### ISSUE TITLE
Broken site audit: TumblrRipper (tumblr.com)

### ISSUE THREAD

**Issue by metaprime** [RipMeApp/ripme#250]
See:

src/main/java/com/rarchives/ripme/ripper/rippers/TumblrRipper.java
src/test/java/com/rarchives/ripme/tst/ripper/rippers/TumblrRipperTest.java

From navigating the site: (NSFW) https://goood-thickness.tumblr.com/ (status code 400 error bad request)

From test:
http://wrouinr.tumblr.com/archive (not supported at all)
http://topinstagirls.tumblr.com/tagged/berlinskaya (status code 400 error bad request)
http://sadbaffoon.tumblr.com/post/132045920789/what-a-hoe (status code 400 error bad request)

Log contains lots of logs like this:

```
2017-11-21 02:42:53,287 DEBUG ripper.AbstractRipper .addURLToDownload() url: http://78.media.tumblr.com/e893555678f48b48d0509cff7b71f82c/tumblr_oz5nzsktKn1wz3uimo1_1280.png, prefix: , subdirectory, referrer: null, cookies: null
2017-11-21 02:42:53,287 DEBUG ripper.AbstractRipper .addURLToDownload() Downloading http://78.media.tumblr.com/e893555678f48b48d0509cff7b71f82c/tumblr_oz5nzsktKn1wz3uimo1_1280.png to .\rips\tumblr_goood-thickness.tumblr.com\tumblr_oz5nzsktKn1wz3uimo1_1280.png
2017-11-21 02:42:53,291 DEBUG ripper.DownloadFileThread .run() Status code: 400
2017-11-21 02:42:53,291 ERROR ripper.DownloadFileThread .run() [!] Non-retriable status code 400 while downloading from http://78.media.tumblr.com/44bb7cc68f8530158788fb1e880293d8/tumblr_ozdxdmKn9O1wz3uimo2_1280.png
```

**Comment by Hrxn** [comment]
I think it's caused by checking for a cookie, to see if restricted content is allowed in the profile.

**Comment by cyian-1756** [comment]
@metaprime I fixed the 400 error issues in release 1.7.8
